roles/nginx/templates/nginx_xnat.j2

```diff
@@ -10,11 +11,11 @@ events {
 
 stream {
     upstream dicom {
-        server localhost:{{ xnat_dicom_port }};
+        server localhost:{{ nginx_upstream_port }};
     }
 
     server {
-        listen {{ dicom_port }};
+        listen {{ nginx_upstream_listen_port }};
```

Comment:
Variable names look fine to me. I wonder whether this role should XNAT agnostic though - maybe the template should live in the `xnat` role and here we provide a basic template of a reverse proxy conf (with TLS termination). This is how I would override for OMERO (rather than adding another conf here).

Reply:
yeah good point, I've tried to change it to make it more general now